src/theme/ods-chart-theme.ts

```diff
@@ -59,12 +59,12 @@ import { isMainAxis } from './const/main-axis-type.const';
 /**
  * ODSChartsCategoricalColorsSet is used for predefined color sets.
  *
- * Default color sets are defined so all the colors are `accessible` against white/dark background and distinguishable for most color-blind people. These sets are not accessible from a monochromacy point of view.
+ * Default and Categorical color sets are defined so all the colors are `accessible` against white/dark background and distinguishable for most color-blind people. These sets are not accessible from a monochromacy point of view.
  *
  * If you use our `sequential` sets, please consider using one color out of two if possible for a greater contrast (for example 0-2-4).
  *
- * - {@link DEFAULT} is the default color set for up to 5 different colors meaning 5 different series.
- * - {@link DEFAULT_LONG} is the default color set for up to 10 different colors grouped by 2. You shouldn't need more in your charts.
+ * - {@link DEFAULT} is the default color set for up to 12 different colors.
+ * - {@link CATEGORICAL} is the default design color set for up to 12 different colors.
```

Comment:
I am not sure users will understand the difference between those two sets ?

What do you think about something like :  
```
 * - {@link DEFAULT} is the default color set for up to 12 different colors, selected to favour contrast differences.
 * - {@link CATEGORICAL} is the set of 12 colours for display by category as defined in the Orange Design System recommendations for dataviz.
```